rtl/eclrtl/eclregex.cpp

```diff
@@ -63,13 +63,13 @@ static void pcre2Free(void * block, void * /*userData*/)
 /// @param regex        OPTIONAL; regex pattern that was in play when error occurred
 static void failOnPCRE2Error(int errCode, const char * msgPrefix, const char * regex = nullptr)
 {
-    int errBuffSize = 120;
+    const int errBuffSize = 120;
     char errBuff[errBuffSize];
     std::string msg = msgPrefix;
     int msgLen = pcre2_get_error_message_8(errCode, (PCRE2_UCHAR8 *)errBuff, errBuffSize);
     if (msgLen > 0)
```

Comment:
Minor:  You can get a negative msgLen and still have a valid error message (for example PCRE2_ERROR_NOMEMORY) albeit truncated suspect you will never encounter that though...

Reply:
Agreed, but the PCRE2 docs say:  _None of the messages are very long; a buffer size of 120 code units is ample._